algosdk/atomic_transaction_composer.py

```diff
@@ -718,7 +718,25 @@ def simulate(
 
         # Parse out abi results
         txn_results = [t["txn-result"] for t in txn_group["txn-results"]]
-        method_results = self.parse_response(txn_results)
+        method_results: List[ABIResult] = []
+        for method_index, method in self.method_dict.items():
+            tx_id = self.tx_ids[method_index]
+            result: ABIResult = ABIResult(
+                tx_id=tx_id,
+                raw_value=b"",
+                return_value=None,
+                decode_error=None,
+                tx_info={},
```

Comment:
`tx_info = txn_results[method_index]`?

Reply:
good point! addressed in [5e2d2b7](https://github.com/algorand/py-algorand-sdk/pull/451/commits/5e2d2b70930d8fe2fc623becc941d743fa5fc1d0)